packages/p2p-v2/src/utils/constants.ts

```diff
@@ -2,3 +2,21 @@ export const COUNTERPARTIES_DROPDOWN_LIST = [
     { value: 'all', text: 'All' },
     { value: 'blocked', text: 'Blocked' },
 ];
+
+export const RATE_TYPE = {
+    FLOAT: 'float',
+    FIXED: 'fixed',
+};
+
+export const AD_ACTION = {
+    EDIT: 'edit',
+    CREATE: 'create',
+    ACTIVATE: 'activate',
+    DEACTIVATE: 'deactivate',
+    DELETE: 'delete',
+};
+
+export const ADVERT_TYPE = {
+    SELL: 'Sell',
+    BUY: 'Buy',
+};
```

Comment:
can we move these constants out to their own individual files? we already have a constants folder, we should move them there, and I had changed and deleted this file when fixing the imports and absolute paths